frontend/src/app/starters/InterestsStarter.tsx

```diff
@@ -1,20 +1,40 @@
 import React from 'react'
 import './InterestsStarter.css'
 import { ConversationStarterProps } from './ConversationStarter'
+import { pluralize } from '../utils'
```

Comment:
There is some funky bug going on with how Storybook handles imports, this import is causing the Storybook tests to not load, but otherwise everything works fine. Don't worry, it's nothing you caused :)

Can you change line 4 to this?

```js
import { pluralize } from '../utils/Utils'
```

Then the Storybook tests will load without error.

Reply:
Or you can leave it, merge now, and I will update it afterwards, since this only affects the test.

Reply:
Did it already, thank you for your feedback!